File: ver0.1.c
//Author: Nathaniel Mugenyi
//Date: 22, July, 2024

//Description:
//1. Convert decimal to hex
//2. Convert hex to decimal


#include <stdio.h>


int main() {

    int ans;// answer

    while(ans != 0){


        int conv; //converting variable

        printf("\nTo convert HEX to DECIMAL enter '1'\nTo convert DECIMAL to HEX enter '2'\nTo exit enter '0'\n");
        scanf("%d", &ans);

        if(ans == 1){ //convert hex to dec
            printf("Enter the HEXADECIMAL: \n");
            scanf("%x",&conv);

            printf("Your Number in DECIMAL: %d",conv);


        }

        else if (ans == 2){ //convert dec to hex
            printf("Enter the DECIMAL: \n");
            scanf("%d",&conv);

            printf("Your Number in HEXADECIMAL: %x",conv);


        }
    }

}

File: ver0.1.py
#Author: Nathaniel Mugenyi
#Date: 01, Aug, 2024

#Description:\/ 0.1
#1.Convert decimal to hex
#2. Convert hex to decimal



ans = None

while (ans != 0):
    ans = int(input("\nTo convert HEX to DECIMAL enter '1'\nTo convert DECIMAL to HEX enter '2'\nTo exit enter '0'\n"))

    if(ans == 1): #convert hex to dec
        conv = str(input("Enter the HEXADECIMAL: \n"))
        print("Your Number in DECIMAL: ", int(conv,16)) #hex(dec) function

    elif (ans == 2): #convert dec to hex
        conv = int(input("Enter the DECIMAL: \n"))
        print("Your Number in HEXADECIMAL: " + hex(conv))
